hatchways/team-eagle
# BE/FE : Feature/Prettier ESLint Configuration

In this pull request, I:
- Installed and configured 'prettier' and 'eslint' for
  - Root project directory
  - `./client` project directory
    - I had to add special configuration to the client since it included React code, and `create-react-app` required special configuration as well.


#### How to test
Installation:
1. `yarn install` in root directory
2. `yarn install` in `./client`
3. `yarn start` in `./client`
  3.1. If this doesn't work, try removing `yarn.lock` and `node_modules` from both root and `./client` directories.
  3.2. Re-install in both directories and `yarn start` in `.client`
4. `yarn dev` in `./server`


Adding double quotes, or tabbing with 4 spaces should now be underlined with red in you VSCode editor.

Any files that change will get automatically prettified on commit. So you don't have to worry about changing every suggested detail that `ESLint` gives. Committing should hopefully be enough.


Please let me know if you get any errors or have any questions
Thanks,
H
